sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -304,9 +311,13 @@ def drive_compilation(self):
             self.vpp_llvm_input = (
                 tmpdir / f"{outstem}_kernels.opt.xpirbc")
             self._asm_ir()
+            # Compilation commands are generated in main process to ensure
+            # they are printed on main process stdout if command dump is set
+            compile_commands = map(self._get_compile_kernel_cmd_out, list(
+                k for k in self.kernel_properties["kernels"]))
```

Comment:
Curious this need for a list interpolation here

Reply:
Indeed, that is not needed.